### DISCUSSION TITLE
ignoring all files in a directory *except* a specific file?

### DISCUSSION THREAD

**Discussion by Dieterbe** [CyberShadow/aconfmgr#187] [Q&A]
I like to use `IgnorePath '/etc/ssl/certs/*'` because i don't need to track all of those certs that are on the system by default.
however, i have added a cert myself, deployed at /etc/ssl/certs/server.crt - i would like to not ignore this specific file.
is this possible? since aconfmgr didn't pick up the file (since it is ignored), i tried manually adding it by copying it to the files directory and using `CopyFile /etc/ssl/certs/server.crt 644` in my config.

However, upon my next `save` run, aconfmgr generates these lines:

```
RemoveFile /etc/ssl/certs/server.crt
RemoveFile /etc/ssl/certs
SetFileProperty /etc/ssl/certs/server.crt mode ''
```

**Comment by seandheath** [comment]
I'm in the same boat, would love a whitelist capability.
